src/components/Mainview/Mainview.css

```diff
@@ -0,0 +1,50 @@
+:root {
+  --component-lenght: 250px;
+}
+
+.main {
+  height: calc(var(--component-lenght)*2);
+  width:100%;
+  background-image: url("https://t4.ftcdn.net/jpg/02/37/82/35/240_F_237823505_hxD7SRiYLVC647y1Pr8vKk8zm3jzkmRI.jpg");
+  background-repeat: no-repeat;
+  background-size: 100%;
```

Comment:
For responsiveness reasons, I'd recommend changing to:
`background-size: cover;
background-position: center;`